packages/pn-personagiuridica-webapp/src/__test__/App.test.tsx

```diff
@@ -85,9 +85,8 @@ describe('App', async () => {
     expect(footer).toBeInTheDocument();
     const sideMenu = result!.queryByTestId('side-menu');
     expect(sideMenu).not.toBeInTheDocument();
-    expect(result!.container).toHaveTextContent(
-      'Non hai le autorizzazioni necessarie per accedere a questa pagina'
-    );
+    const loginFePage = result!.queryByTestId('access-denied');
```

Comment:
Remove this check and verify the the window.open is called once and with login url